halo2-ecc/src/secp256k1/tests/schnorr_signature.rs

```diff
@@ -57,8 +57,7 @@ pub fn schnorr_signature_test<F: BigPrimeField>(
 pub fn random_schnorr_signature_input(rng: &mut StdRng) -> SchnorrInput {
     let sk = <Secp256k1Affine as CurveAffine>::ScalarExt::random(rng.clone());
```

Comment:
The point of the random test is to use different seeded Rngs.  So you should use the given `rng` but mutate it.

Since `rng: &mut StdRng` is already a mutable reference, I think you should replace all `rng.clone()` with just `rng`.